src/components/nodeColors.js

```diff
@@ -1,6 +1,72 @@
+import Color from 'color';
+
+export const baseNodeColors = [
+  '#357bf6',
+  '#f6c243',
+  '#6d747c',
+  '#4ba0b5',
+  '#53a451',
+  '#cc444a',
+];
+
 export const validNodeColor = '#dffdd0';
 export const invalidNodeColor = '#fae0e6';
+
 export const defaultNodeColor = '#fff';
+export const defaultNodeColorStroke = '#000';
+
 export const startColor = '#edfffc';
+export const startColorStroke = '#00bf9c';
+
+export const intermediateColor = '#fff4d1';
+export const intermediateColorStroke = '#fbbe02';
+
 export const endColor = '#fff1f2';
+export const endColorStroke = '#ed4757';
+
 export const poolColor = '#f7f7f7';
+
+export function getDefaultNodeColors(node, color) {
+  if (color) {
+    return { fill: Color(color).lighten(0.3).hex(), stroke: color };
```

Comment:
This is great - I've noticed that the contrast between text and the grey is not super high, so if you want consider upping the lightening to something like 40% or so